code/modules/mob/living/carbon/life.dm

```diff
@@ -735,27 +735,29 @@ All effects don't start immediately, but rather get worse over time; the rate is
 //LIVER//
 /////////
 
-///Decides if the liver is failing or not.
+///Check to see if we have the liver, if not automatically gives you last-stage effects of lacking a liver.
```

Comment:
I am unsure whether it makes sense to duplicate the stage 5 failure functionality, both in the liver, and in on_life.

Reply:
well we cant really call a function of an object that doesnt exist, and this is the simplest fix.

Reply:
Ideally the liver would serve some function and the issue would come from having it not do that. The issue is that mobs are currently too disassociated for that to be possible.

Reply:
besides, if someone will have a better idea as to what to put there i will implement it , for now it can stay.

Reply:
This could be very easily solved if we had something like a static function, but these dont exist in DMM.

Reply:
> 
> 
> Ideally the liver would serve some function and the issue would come from having it not do that. The issue is that mobs are currently too disassociated for that to be possible.

liver currently is just the thing that makes things metabolize, effect due to lack of it have to be more than just 'now you dont process chems'